src/sass/extends/_headings.scss

```diff
@@ -74,6 +73,14 @@ h5 {
   letter-spacing: $heading-five-letter-spacing;
 }
 
+h1 span[class^='highlight-'],
```

Comment:
- Multiple fixes since last update. See everything above May 10 here: https://github.com/ASU/asu-unity-stack/commits/dev/packages/bootstrap4-theme/src/scss/extends/_headings.scss
- Looks like they were solving padding & spacing issues when a highlighted header occurs after a paragraph. 
- Looks OK to merge. Markup for these changes not affected. 
- Worth noting that this work was done specifically within the context of highlights in a hero image.